web/src/screens/roleShow.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {Container, Col, Row} from 'react-bootstrap'
+import {jobShowHeader as Header, jobShowCarousel as JobGraphics, jobShowSummary as Summary } from '../components/Cards'
+
+const style = {
+    header : "card-subtitle center my-2 md-6",
+    graphics: "",
+    summary: "card-title center md-6 my-3"
+}
+const imgArr = [
```

Comment:
Don't use Hungarian notation. We know it's an array because... it's an array. `images` is clearer for an array of images.